NeZha-D1s/README.md

```diff
@@ -4,22 +4,27 @@
 
 ### 操作系统信息
 
-- Tina Linux
-  - 下载链接：链接：https://pan.baidu.com/s/1v55AKMFripaEu22tJ92lmw?pwd=awol 提取码：awol
+- Tina Linux:
+  - 下载链接: https://pan.baidu.com/s/1v55AKMFripaEu22tJ92lmw?pwd=awol 提取码：awol
   - 参考安装文档：https://d1s.docs.aw-ol.com/study/study_1tina/
 - RT-Thread:
   - 下载链接：https://github.com/RT-Thread/rt-thread
   - 参考安装文档：https://github.com/RT-Thread/rt-thread/blob/master/bsp/allwinner/d1s/README-M7.md
+  - Ubuntu:
+  - 下载链接：https://cdimage.ubuntu.com/releases/noble/release/ubuntu-24.04-preinstalled-server-riscv64+nezha.img.xz?_gl=1*1x36sbc*_gcl_au*NDI5ODM0MDYxLjE3MTgyNjU3NzE.&_ga=2.251705994.2059181571.1718425068-1272776944.1718425068
```

Comment:
Why so much garbage after .xz?
I assume these are for tracking / data collection / metrics. Better remove them.